ds701_book/25-NN-III-CNNs.qmd

```diff
@@ -372,124 +377,121 @@ Problems with fully-connected networks
 * Should be stable under transformations
```

Comment:
This sentence does not make sense to me. How is it a problem? If they should be stable under transformations wouldn't that be a good thing?

Reply:
that was referring to FCNs. The slide title was incorrect. I changed it and moved it to the beginning of the section in the next commit.